### PR TITLE
Create GH action for tests

### PR DESCRIPTION
**Pull request by tommyod**

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1,5 +1,3 @@
-# List of requirements
-# Run 
-sympy>=1.1.1
-numpy>=1.13.1
+sympy
+numpy
```
Comment (VascoSch92) [inline requirements.txt:2]: are you sure that you don't want to control the versions here?

File: .github/workflows/python-package.yml [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+        pip install --upgrade pip
+        pip install flake8 pytest sympy
+    - name: Install package
+      run: |
+        pip install -r requirements.txt
+        pip install -r test-requirements.txt
+        pip install .
+    - name: Test with pytest
+      run: |
+        pytest
```
Comment (VascoSch92) [inline .github/workflows/python-package.yml:38]: i'm totally not an expert for GitHub actions but it seems there is a problem with `sympy` not installed. Is it in the `test-requirements.txt` too ? 

You can also give a look to the GitHub Actions docs for Python. It seems quite good: https://docs.github.com/en/actions/automating-builds-and-tests/building-and-testing-python

**Comment by VascoSch92** [comment]
nice ;-) good job

**Comment by VascoSch92** [comment]
why you cannot just `from abelian import __version__` ?

**Comment by tommyod** [comment]
It triggers imports further down in the package. But the package is not installed yet! So we run into trouble.

**Pull request merged**